tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "@stellar/tsconfig",
   "compilerOptions": {
     "importHelpers": true,
-    "target": "es5",
+    "target": "ES2019",
```

Comment:
Given our runtime (modern Chrome, Firefox on the horizon) we can target a much more recent version of JS without needing to transpile down. A big change here is that compiled output will natively use async/await, which should make debugging easier